sdk/documenttranslator/ai-document-translator-rest/review/ai-document-translator.api.md

```diff
@@ -4,738 +4,812 @@
 
 ```ts
 
+import type { AbortSignalLike } from '@azure/abort-controller';
+import type { CancelOnProgress } from '@azure/core-lro';
 import type { Client } from '@azure-rest/core-client';
 import type { ClientOptions } from '@azure-rest/core-client';
+import type { CreateHttpPollerOptions } from '@azure/core-lro';
 import type { HttpResponse } from '@azure-rest/core-client';
 import type { KeyCredential } from '@azure/core-auth';
+import type { OperationState } from '@azure/core-lro';
+import type { PagedAsyncIterableIterator } from '@azure/core-paging';
+import type { PathUncheckedResponse } from '@azure-rest/core-client';
 import type { RawHttpHeaders } from '@azure/core-rest-pipeline';
 import type { RequestParameters } from '@azure-rest/core-client';
-import type { TokenCredential } from '@azure/core-auth';
+import type { StreamableMethod } from '@azure-rest/core-client';
 
-// @public (undocumented)
+// @public
 export interface BatchRequest {
     source: SourceInput;
-    storageType?: StorageInputType;
-    targets: TargetInput[];
+    storageType?: "Folder" | "File";
```

Comment:
why are we not generating the type alias here?

Reply:
I think this should be expected?